python/cloudtik/runtime/spark/scripts/configure.sh

```diff
@@ -114,6 +116,10 @@ function update_credential_config_for_aws() {
     if [ "$AWS_WEB_IDENTITY" == "true" ]; then
         # Replace with InstanceProfileCredentialsProvider with WebIdentityTokenCredentialsProvider for Kubernetes
         sed -i "s#InstanceProfileCredentialsProvider#WebIdentityTokenCredentialsProvider#g" `grep "InstanceProfileCredentialsProvider" -rl ./`
+        WEB_IDENTITY_ENVS="spark.yarn.appMasterEnv.AWS_ROLE_ARN   ${AWS_ROLE_ARN}\nspark.yarn.appMasterEnv.AWS_WEB_IDENTITY_TOKEN_FILE   ${AWS_WEB_IDENTITY_TOKEN_FILE}\nspark.executorEnv.AWS_ROLE_ARN   ${AWS_ROLE_ARN}\nspark.executorEnv.AWS_WEB_IDENTITY_TOKEN_FILE   ${AWS_WEB_IDENTITY_TOKEN_FILE}"
```

Comment:
Will empty "${AWS_ROLE_ARN}" and "${AWS_WEB_IDENTITY_TOKEN_FILE}" value cause problem?
If not, it OK to do this.